usaspending_api/download/delta_models/account_balances_download.py

```diff
@@ -170,7 +172,9 @@ def account_balances_df(spark: SparkSession) -> DataFrame:
 
 def load_account_balances(spark: SparkSession, destination_database: str, destination_table_name: str) -> None:
     df = account_balances_df(spark)
-    df.write.format("delta").mode("overwrite").saveAsTable(f"{destination_database}.{destination_table_name}")
+    df.write.format("delta").mode("overwrite").partitionBy(
```

Comment:
Should verify, but I believe the data should be partitioned upon writing to the Delta table since the Delta table itself is partitioned. When testing locally I was able to define the partitioned at table creation and then run a simple `INSERT INTO` and see data partitioned.

Did you come across anything that needed the partitions defined on the DataFrame Writer?